src/app/[lang]/partners/page.tsx

```
@@ -1,3 +1,26 @@
+
+  return (
+    <main className={`flex flex-col gap-8 sm:gap-12 ${horizontalPadding + verticalPadding}`}>
+      <div className="grid grid-cols-1 gap-2.5 sm:gap-3 md:grid-cols-2 lg:grid-cols-3">
+        {partners.map((partner) => (
+          <PartnerCard
+            title={partner.title}
+            url={partner.url}
+            logo={partner.logo}
+            color={partner.color}
+            perks={partner.perks}
+          />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)